core-libs/setup/ssr/providers/ssr-providers.ts

```diff
@@ -35,6 +38,11 @@ export function provideServer(options?: ServerOptions): StaticProvider[] {
       provide: LoggerService,
       useFactory: serverLoggerServiceFactory,
     },
+    {
+      provide: MULTI_ERROR_HANDLERS,
+      useClass: ServerRespondingErrorHandler,
```

Comment:
For better extensibility (to allow for customizing `ServerRespondingErrorHandler`):
```suggestion
      useExisting: ServerRespondingErrorHandler,
```

Reply:
Thanks, good catch! I used `useExisting` before but I changed it to test something out. Forgot to revert the change.